css/src/cleantalk-trp.css

```diff
@@ -65,5 +65,5 @@
 
 .apbct-real-user-popup-header {
     font-weight: bolder;
-    margin: 0;
+    margin: 2px!important;
```

Comment:
`2px!important` - Is it need a space between `2px` and `!important`?

Reply:
You are right, with space is prefered. Fix https://github.com/CleanTalk/wordpress-antispam/pull/671/commits/79135745b52c7f0ab9ad4029ef87186c4becb46e